packages/neon_lints/lint_maker.yaml

```diff
@@ -26,4 +11,18 @@ dart:
         todo: ignore
       exclude:
         - '**.g.dart'
-        - '**.mocks.dart'
+    linter:
+      rules:
+        - always_specify_types
+        - avoid_annotating_with_dynamic
+        - avoid_as
+        - avoid_catches_without_on_clauses
+        - avoid_final_parameters
+        - avoid_print
+        - diagnostic_describe_all_properties
+        - lines_longer_than_80_chars
+        - no_default_cases
+        - one_member_abstracts
+        - prefer_double_quotes
+        - prefer_relative_imports
+        - unnecessary_final
```

Comment:
Shouldn't those be set to `: false`?

Reply:
It can be debated wether this makes sense but all rules listed in the lint_maker.yaml file are considered to not be wanted thus disabled.
I wrote it that way because that's the only thing that makes sense in the context of enabling all available rules by default.

The syntax with `rule_name: false` is also possible. Should I change it over here?

Reply:
I mean it is totally unintuitive because the key is called rules and not disabled_rules or something like that.

Reply:
I initially wanted to keep the syntax 1:1 compatible to the analysis_options file to leverage a parser from the analyzer package which I didn't end up doing.

Do you think renaming the map to `disabled_rules` as you mentioned is better or should I just use the analysis_options syntax and interpret list elements as enabled?